extensions/Worker.Extensions.EventGrid/src/TypeConverters/EventGridHelper.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Microsoft.Azure.Functions.Worker.Converters;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.EventGrid.TypeConverters
+{
+    internal static class EventGridHelper
+    {
+        internal static ValueTask<ConversionResult> ConvertHelper(ConverterContext context)
```

Comment:
I think we can rename this to be more specific to what this function is doing i.e `DeserilizeToTargetType`

I wonder if we want to leave each converter in control of the ConversionResult and we should just have this helper return whatever `item` is?

Reply:
Discussed offline and decided to keep structure